engine/src/main/scala/cromwell/engine/workflow/tokens/JobTokenDispenserActor.scala

```diff
@@ -48,19 +49,21 @@ class JobTokenDispenserActor(override val serviceRegistryActor: ActorRef,
   }
 
   // Give the actor time to warm up, then start scheduling token allocation logging:
-  context.system.scheduler.scheduleOnce(5.seconds) {
-    effectiveLogInterval match {
-      case Some(someInterval) =>
-        log.info(s"Triggering log of token queue status. Effective log interval = $someInterval")
-        context.system.scheduler.scheduleOnce(someInterval) {
-          self ! LogJobTokenAllocation(someInterval)
-        }(context.dispatcher)
-        ()
-      case None =>
-        log.info(s"Not triggering log of token queue status. Effective log interval = None")
-        ()
-    }
-  }(context.dispatcher)
+  if (logTokenAllocation) {
```

Comment:
This was already optional - see the `lazy val effectiveLogInterval: Option[FiniteDuration] = logInterval.filterNot(_ == 0.seconds)` safeties above (and the default was already 0, ie off). If you want to change from "flag value" to "explicit log or not boolean" then sure... but I'd wrap that logic up into the `effectiveLogInterval` value rather than re-checking it here.